src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_ArrayCreationAndInitializer.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
-    public partial class IOperationTests : SemanticModelTestBase
+    public class IOperationTests_ArrayCreationAndInitializer : SemanticModelTestBase
```

Comment:
If it's not too much work, I'd _prefer_ if we named these things something like `ArrayCreationAndInitializerOperationTests`.

Reply:
The file can be renamed in a follow-up.

Reply:
Alright. That's why I left it at prefer.